Smartpad-iOS/Smartpad-iOS/Controller/MainViewController.swift

```diff
@@ -288,11 +288,13 @@ class MainViewController: UIViewController {
 extension MainViewController: UIGestureRecognizerDelegate {
     func gestureRecognizer(
       _ gestureRecognizer: UIGestureRecognizer,
-      shouldRecognizeSimultaneouslyWith simultaneousGestureRecognizer: UIGestureRecognizer
+      shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer
     ) -> Bool {
-        if ((gestureRecognizer == self.singleTouchGestureRecognizer || gestureRecognizer == self.doubleTouchGestureRecognizer) && (simultaneousGestureRecognizer == self.singleTapGestureRecognizer || simultaneousGestureRecognizer == self.singleTapDoubleClickGestureRecognizer || simultaneousGestureRecognizer == self.doubleTapGestureRecognizer || simultaneousGestureRecognizer == self.singlePanGestureRecognizer || simultaneousGestureRecognizer == self.doublePanGestureRecognizer || simultaneousGestureRecognizer == self.dragPanGestureRecognizer || simultaneousGestureRecognizer == self.pinchGestureRecognizer)) {
+
+        /* Only allow simultaneous gestures when one of the gestures is singleTouch or doubleTouch */
+        if (gestureRecognizer == self.singleTouchGestureRecognizer || gestureRecognizer == self.doubleTouchGestureRecognizer) {
```

Comment:
@ArthurChan0427 Do you see any problem with this? I read it as "allow multiple gestures if one of the gestures in the single touch or double touch gesture". In this case, we can just drop the && portion of the check

Reply:
This is fine, assuming we aren't making any special cases where two gestures are allowed to triggered simultaneously.